akka-distributed-data/src/main/scala/akka/cluster/ddata/GCounter.scala

```diff
@@ -65,14 +65,19 @@ final class GCounter private[akka] (
    * Increment the counter with the delta `n` specified.
    * The delta must be zero or positive.
    */
-  def +(n: Long)(implicit node: Cluster): GCounter = increment(node, n)
+  def :+(n: Long)(implicit node: SelfUniqueAddress): GCounter = increment(node.uniqueAddress, n)
+
+  @deprecated("Use `:+` that takes a `SelfUniqueAddress` parameter instead.", since = "2.5.19")
+  def +(n: Long)(implicit node: Cluster): GCounter = increment(node.selfUniqueAddress, n)
 
   /**
    * Increment the counter with the delta `n` specified.
    * The delta `n` must be zero or positive.
    */
-  def increment(node: Cluster, n: Long = 1): GCounter =
-    increment(node.selfUniqueAddress, n)
+  def increment(node: SelfUniqueAddress, n: Long): GCounter = increment(node.uniqueAddress, n)
```

Comment:
`n: Long = 1`? or do we prefer to not have a default value?

Reply:
Default values can introduce bugs as things change. For backwards compat I left the deprecated one as is.